backend/bin/provision.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -e
+
+BIN_DIR=$(dirname "$0")
+source "${BIN_DIR}/config.sh"
+  
+ssh "$SERVER_USER@$SERVER_HOST" <<EOF
+  set -e
+  
+  mkdir -p wotr-server
+  
+  echo "Acquiring SSL certificate..."
```

Comment:
I tried to figure out a script for automating a letsencrypt SSL cert renewal at one point in my life and remember just giving up in despair lol, this script is sweet and I will probably reference it in the future

Reply:
Lemme know if you wanna chat about any of the UNIX-y stuff!

Reply:
🙏